plugins/woocommerce-admin/client/activity-panel/activity-panel.js

```diff
@@ -298,9 +298,27 @@ export const ActivityPanel = ( { isEmbedded, query } ) => {
 			},
 		};
 
-		return [ activity, setup, previewSite, displayOptions, help ].filter(
-			( tab ) => tab.visible
-		);
+		const previewStore = {
+			name: 'previewStore',
+			title: __( 'Preview store', 'woocommerce' ),
+			icon: <Icon icon={ external } />,
+			visible: isHomescreen(),
+			onClick: () => {
+				window.open( getSetting( 'siteUrl' ) );
```

Comment:
Mine redirects to `/wp-admin/false`, because `getSetting` returns `false`.
I believe you want to make use of `getAdminSetting` or do `getSetting('admin').siteUrl`.

But instead of `siteUrl` I think you want to use `shopUrl`? given `siteUrl` just goes to the home page of the site, not the store.
`shopUrl` is the same url that is used when hovering over `WooCommerce <site name>` in the top left and clicking  `Visit store`.

Reply:
Nice catch! I fixed the link in commit 93b9c48